docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -0,0 +1,176 @@
+                --sysctl "net.ipv6.conf.all.disable_ipv6=0 net.ipv4.conf.all.forwarding=1 net.ipv6.conf.all.forwarding=1" \
+                --volume /dev/rcp-dk:/dev/ttyACM0 openthread/otbr --radio-url spinel+hdlc+uart:///dev/ttyACM0
+3. Open `http://localhost:8080/` address in a web browser.
+4. Click at the __Form__ button on the left menu.
+5. Notice that On-Mesh Prefix is equal to `fd11:22::`. It will be used later to configure IPv6 packet routing.
+6. Click at the __Form__ button at the bottom to form a new Thread network using the default settings.
+7. Check the output from `docker network inspect otbr` to learn which IPv6 address was assigned to the OTBR Docker container. Most likely, it will be `2001:db8:1::2`.
+8. Run the following command, with _otbr-container-addr_ replaced with the address retrieved in the previous step, to ensure that packets targeting Thread network nodes are routed via the OTBR Docker container:
+
+        $ sudo ip -6 route add fd11:22::/64 dev otbr0 via otbr-container-addr
+
+<br>
+To form a Wi-Fi network, perform the following steps:
+
+1. Open Ubuntu settings widget, by running the command:
+
+        $ gnome-control-center
+2. Navigate to Wi-Fi settings.
+3. Click at the 3-dot icon at the title bar and select __Turn On Wi-Fi Hotspot...__ option.
+4. Enter a network name and password and click at the __Turn On__ button.
+5. Run the following command to assign a well-known IPv6 address to the hotspot interface:
+
+        $ nmcli connection modify Hotspot ipv6.addresses 2001:db8:2::/64
+6. Install Routing Advertisement Daemon to enable IPv6 auto-configuration of devices connecting to the hotspot:
+
+        $ sudo apt install radvd
+7. To learn the hotspot interface name:
+
+        $ nmcli connection show Hotspot | grep interface-name  
+8. Put the following content with _wlo1_ replaced with the hotspot interface name in `/etc/radvd.conf`:
+
+        interface wlo1
+        {
+        MinRtrAdvInterval 3;
+        MaxRtrAdvInterval 4;
+        AdvSendAdvert on;
+        AdvManagedFlag on;
+        prefix 2001:db8:2::/64
+        { AdvValidLifetime 14300; AdvPreferredLifetime 14200; };
+        };
+9. Start `radvd` service:
+
+        $ systemctl start radvd
+
+</details>
+
+## Building and flashing nRF Connect Lock Example Application
+
+Read [nRF Connect Lock Example Application README](../../examples/lock-app/nrfconnect/README.md) to learn how to build and flash the lock firmware onto nRF 52840 DK.
+
+## Building and installing Android CHIPTool
+
+<details>
+<summary>Click here to expand</summary>
+<br>
+
+1. Read [Android CHIPTool README](../../src/android/CHIPTool/README.md) to learn how to build the application for your smartphone.
+2. Install `android-tools-adb` package:
+
+        $ sudo apt install android-tools-adb
+3. Enable USB debugging on the device by following the [article](https://developer.android.com/studio/debug/dev-options).
```

Comment:
Install via USB option also should be enabled.

Reply:
Isn't it enabled by default when you enable USB debugging? I can't find that option in my Android....

Reply:
Oh I see, it seems that it's some Xiaomi specific option, as in the Internet I can find articles applying only for this smartphone brand. I got one and after enabling USB debugging I can read logs from device, but not install apps (when trying to upload, it fails with information that installing is locked). I think that you can ignore it, as it doesn't apply to all Android devices.

Reply:
Actually I added that the option should be turned on if it's present. It's good to cover different devices. Thanks for spotting that!